src/webapp/src/auth/Auth.tsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import styled from 'styled-components';
 import { useDispatch, useSelector } from 'react-redux';
 import { AuthStatuses } from './types';
-import { RootState } from '../reducers';
 import { signInUser, signOutUser } from './asyncAuthActions';
+import { selectUser } from "./selectors"
```

Comment:
```suggestion
import { selectUser } from './selectors'
```
double quotes arent what we use usually, what makes us want to have them here?

Reply:
I have no idea why they are appearing there! It is driving me crazy, I have singleQuotes: true in every possible field.